### PR TITLE
Add tests for ERC20 and ERC721 contract calls

### PR DESCRIPTION
**Pull request by nmlinaric**
## Description
Added tests for ERC20 and ERC721 contract calls

## Related Issue Or Context
Closes: #209 

## How Has This Been Tested? Testing details.
Added unit tests

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Comment by MakMuftic** [comment]
@nmlinaric this PR looks good, but I think we need similar unit tests for `bridge` and `assetStore` contracts. I think it is fine to split them into issues if you want to merge this first.

**Comment by nmlinaric** [comment]
> @nmlinaric this PR looks good, but I think we need similar unit tests for `bridge` and `assetStore` contracts. I think it is fine to split them into issues if you want to merge this first.

Agree,I'll add them to this PR.

**Review by MakMuftic — APPROVED** [review]
Looks good, just this small remark regarding test naming.

File: chains/evm/calls/contracts/bridge/bridge_test.go [hunk 1]
```diff
@@ -35,10 +47,19 @@ func (s *ProposalStatusTestSuite) SetupTest() {
...
+		testResourceId,
+		[]byte{},
+		common.HexToAddress(testHandlerAddress),
+		s.bridgeAddress,
+	)
 }
 func (s *ProposalStatusTestSuite) TearDownTest() {}
 
-func (s *ProposalStatusTestSuite) TestProposalStatusFailedContractCall() {
+func (s *ProposalStatusTestSuite) TestProposalStatusFailed_ContractCall() {
```
Comment (MakMuftic) [inline chains/evm/calls/contracts/bridge/bridge_test.go:62]: Could you just rename these old tests to reflect the same name structure as new tests as I really like how you structured function (test) names.

**Review by MakMuftic — APPROVED** [review]

**Review by P1sar — APPROVED** [review]

**Pull request merged**